app/products/calls/state/actions.ts

```diff
@@ -61,6 +70,120 @@ export const setCalls = (serverUrl: string, myUserId: string, calls: Dictionary<
     setCurrentCall(nextCall);
 };
 
+export const processIncomingCalls = async (serverUrl: string, calls: Call[], keepExisting = true) => {
+    if (!getCallsConfig(serverUrl).EnableRinging) {
+        return;
+    }
+
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return;
+    }
+
+    // Do we have incoming calls we should notify about?
+    const incomingCalls = getIncomingCalls().incomingCalls;
+    const existingCalls = getCallsState(serverUrl).calls;
+    const myUserId = getCallsState(serverUrl).myUserId;
+    const newIncoming: IncomingCallNotification[] = [];
+
+    for (const call of calls) {
```

Comment:
```suggestion
    for await (const call of calls) {
```

then you can get rid of the `eslint-disable-line`

Reply:
Awesome, didn't know about that, thanks.